inst/tutorials/chapter4/lm_example.Rmd

```diff
@@ -134,8 +134,8 @@ surf <- acast(df, avginc ~ str)
 
 Caschool %>%
   plot_ly() %>%
-  add_markers(x = ~str, y = ~avginc, z = ~testscr, name = "Data", hoverinfo = "skip") %>%
-  add_surface(x = to_plot_x, y = to_plot_y, z = ~surf, inherit = F, name = "Best Fit Plane") %>%
+  add_markers(x = ~str, y = ~avginc, z = ~testscr, name = "Data", hoverinfo = "skip", opacity = .75, marker=list(color = 'green', size = 5)) %>%
```

Comment:
this is *beautiful*. :100:
* i am not sure the color scale on the surface is helpful. i would try removing it.
* anyway, colorbar on the right is not needed
* i would take another color for the points. not sure if red or black. but decrease opacity even further. 0.5?
* so my preferred picture would have a grey opaque plane and red opaque points, even a bit smaller.

Reply:
gotcha

Reply:
done